stories/animatedDots.jsx

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+
+import AnimatedDots from 'components/spinners/animatedDots';
+import { radios, withKnobs } from '@storybook/addon-knobs/src/index';
+
+
+const appearanceKnob = () => radios('Appearance', ['light', 'dark'], 'light');
+
+
+const stories = storiesOf('Animated dots', module).addDecorator(withKnobs);
+
+stories.add('Animated dots', () => (
+  <div style={{
+    background: '#121212',
```

Comment:
not blocking, but this background could change based on the knob being light or dark!